html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationWrixLocations.js

```diff
@@ -0,0 +1,1019 @@
+import i18n from '@/utils/locale'
+import pfFormChosen from '@/components/pfFormChosen'
+import pfFormInput from '@/components/pfFormInput'
+import pfFormRangeToggle from '@/components/pfFormRangeToggle'
+import {
+  pfConfigurationListColumns,
+  pfConfigurationListFields
+} from '@/globals/configuration/pfConfiguration'
+import {
+  and,
+  not,
+  conditional,
+  hasWrixLocations,
+  wrixLocationExists
+} from '@/globals/pfValidators'
+
+const {
+  maxLength,
+  required
+} = require('vuelidate/lib/validators')
+
+export const pfConfigurationWrixLocationsLogs = [
```

Comment:
Same thing as `pfConfigurationSyslogForwardersLogs`? Can we define this array globally? Will be replaced by the new *OPTIONS* API endpoints?

Reply:
This is just a temporary placeholder. It's expected with OPTIONS endpoint(s).